pandas/_libs/lib.pyi

```diff
@@ -25,6 +25,9 @@ class NoDefault(Enum): ...
 
 no_default: NoDefault
 
+i8max: int
```

Comment:
i still find this odd, pls do in a followup (e.g. change these to the same naming as in util e.t. INT64_MAX)